consumer/core/src/test/java/org/arquillian/pact/consumer/core/publisher/FolderPactsPublisherTest.java

```diff
@@ -21,11 +22,12 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class FolderPublishPactsTest extends AbstractManagerTestBase {
+@NotThreadSafe
+public class FolderPactsPublisherTest extends AbstractManagerTestBase {
```

Comment:
We still have copy-paste in those two methods. Can we extract it and maybe even use `@Before` and `@After`?